package.json

```diff
@@ -14,7 +14,9 @@
     "prebuild": "rimraf lib",
     "build": "npm run build:js && npm run build:scss",
     "build:js": "NODE_ENV=production babel ./src --out-dir ./lib --ignore spec.js,test.js,stories.js",
-    "build:scss": "cpx -p './src/styles/**/*.scss' ./lib/styles",
+    "mastersass-styles": "node-sass --output-style compressed src/styles/index.scss dist/css/mastersass.css",
+    "react-styles": "cpx -p './src/styles/**/*.scss' ./lib/styles",
+    "build:scss": "npm run react-styles ; npm run mastersass-styles",
```

Comment:
Use `&&` to chain scripts instead of `;`

`"build:scss": "npm run react-styles && npm run mastersass-styles",`

Reference: https://stackoverflow.com/questions/39172536/running-npm-scripts-sequentially